lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -88,7 +88,10 @@ class TraceOfTab extends ComputedArtifact {
       firstMeaningfulPaint = lastCandidate;
     }
 
-    const onLoad = keyEvents.find(e => e.name === 'MarkLoad' && e.ts > navigationStart.ts);
+    const onLoad = frameEvents.find(e => e.name === 'loadEventStart' && e.ts > navigationStart.ts);
+    const domContentLoaded = frameEvents.find(
+      e => e.name === 'domContentLoadedEventStart' && e.ts > navigationStart.ts
```

Comment:
for both this and `loadEvent`, usually the `End` is used, but in this case, it actually makes more sense for us to focus on the browser's POV of these events... so 👍  on using `Start`